app/_containers/PostCard/index.module.scss

```diff
@@ -0,0 +1,112 @@
+@use "@/modules/styles/_mixins.scss" as mixins;
+
+
+/* Article */
+.article {
+    position: relative;
+    background-color: #ffffff;
+    border-radius: 8px; 
+    overflow: hidden; 
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1); 
+    transition: box-shadow 0.2s; 
+    cursor: pointer;
+}
+  
+  .article:hover {
+    box-shadow: 0 6px 16px rgba(0, 0, 0, 0.15); 
+  }
+  
+/* Image Container */
+.cardImg {
+    position: relative;
+    aspect-ratio: 4 / 3; 
+    overflow: hidden;
+  }
+  
+/* Image */
+.img {
+    object-fit: cover; 
+    width: 100%; 
+    height: 100%; 
+    transition: transform 0.3s; 
+}
+  
+.img:hover {
+    transform: scale(1.05); 
+}
+  
+/* Gradient Overlay */
+.gradient_overlay{
+    position: absolute;
+    inset: 0; 
+    background: linear-gradient(to top, rgba(0, 0, 0, 0.6), transparent);
+}
+  
+/* Title */
+.title_container {
+    position: absolute;
+    bottom: 16px; 
+    left: 16px; 
+    right: 16px; 
+    color: #ffffff; 
+  }
+
+.title_text{
+    font-size: 1.5rem; 
+    font-weight: 600; 
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+  
+/* Tags Container */
+.tags_container {
+    display: flex; 
+    gap: 8px;
+    margin-bottom: 12px; 
+}
+  
+/* Tag */
+.tag {
+    padding: 4px 8px;
+    font-size: 0.875rem; 
+    font-weight: bold;
+    background-color: #f5f5f5; 
+    color: #374151; 
+    border-radius: 9999px;
+    cursor: pointer; 
+    transition: background-color 0.2s; 
+}
+  
+.tag:hover {
+    background-color: #e5e7eb; 
+}
+  
+/* Description */
+.description{
+    color: #4b5563; 
+    margin-bottom: 12px; 
+    display: -webkit-box;
+    -webkit-line-clamp: 2; 
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+  
+/* Time */
+.time {
+    font-size: 0.875rem; 
+    color: #6b7280; 
```

Comment:
should use `colors.$black80` (or something similar to #6b7280)

I defined all the necessary color in one place. We should use them most of the time to improve readability.